src/azure-cli/HISTORY.rst

```diff
@@ -68,6 +68,28 @@ Release History
 **ServiceFabric**
 
 * az sf cluster create: fix #10916 modify service fabric linux and windows template.json compute vmss from standard to managed disks
+* Adding new commands to manage appliaction and services.
```

Comment:
You need to rebase dev branch and move your release notes to the new release notes part, which is above 2.0.76